src/components/safe-apps/SafeAppsSection/CollapsibleSection.tsx

```diff
@@ -0,0 +1,81 @@
+import Accordion from '@mui/material/Accordion'
+import AccordionSummary from '@mui/material/AccordionSummary'
+import AccordionDetails from '@mui/material/AccordionDetails'
+import Typography from '@mui/material/Typography'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import Grid from '@mui/material/Grid'
+import { AppCard } from '@/components/safe-apps/AppCard'
+import { SectionProps } from './types'
+import { AddCustomAppCard } from '@/components/safe-apps/AddCustomAppCard'
+
+const CollapsibleSection = ({
+  title,
+  apps,
+  onPinApp,
+  prependAddCustomAppCard,
+  onAddCustomApp,
+  pinnedIds,
+  cardVariant,
+}: SectionProps) => {
+  const columnSpacing = cardVariant === 'compact' ? 3 : 2
+
+  return (
+    <Accordion
+      sx={{
+        boxShadow: 'none',
+        border: 'none',
+        background: 'transparent',
+      }}
+      defaultExpanded
+    >
+      <AccordionSummary
+        expandIcon={<ExpandMoreIcon />}
+        sx={({ spacing }) => ({
+          padding: `0 ${spacing(3)}`,
+          justifyContent: 'flex-start',
+          background: 'transparent',
+          '& > .MuiAccordionSummary-content': {
+            flexGrow: 0,
+            '&.Mui-expanded': {
+              margin: `${spacing(2)} 0`,
+            },
+          },
+          '&.Mui-expanded': {
+            minHeight: 48,
```

Comment:
Please move to CSS.

Reply:
I've tried to move to CSS, but unfortunately, the styles that overwrite default MUI styles aren't being applied because of `css specificity`. I can't think of a good solution except adding `!important`, but that sounds a little hacky. Do you have a tip?

Reply:
Wasn't aware of `:global` thing